src/main/java/oc/wh40k/units/im/IMDominionSquad.java

```diff
@@ -2,38 +2,43 @@
 
 import oc.*;
 
+import static oc.KeyWord.*;
+
 public class IMDominionSquad extends Eintrag {
 
     AnzahlPanel squad;
-    OptionsZaehlerGruppe o1;
-    OptionsUpgradeGruppe o2;
-    RuestkammerStarter rkBoss;
-    RuestkammerStarter rkTransport;
+    OptionsZaehlerGruppe bolters;
+    OptionsZaehlerGruppe special;
+    OptionsEinzelUpgrade simulacrum;
 
     public IMDominionSquad() {
+    	super(IMPERIUM, ADEPTUS_MINISTORUM, ADEPTA_SORORITAS, ORDER, INFANTRY, DOMINION_SQUAD);
         name = "Dominion Squad\n";
         grundkosten = 0;
         überschriftSetzen = true;
 
         squad = new AnzahlPanel(ID, randAbstand, cnt, "Dominion Squad", 5, 10, getPts("Dominion Squad"));
         add(squad);
 
-        add(ico = new oc.Picture("oc/wh40k/images/ASSororitastrupp.jpg"));
-
         seperator();
 
-        ogE.addElement(new OptionsGruppeEintrag("Storm bolter", getPts("Storm bolter (AMI)")));
-        ogE.addElement(new OptionsGruppeEintrag("Flamer", getPts("Flamer (AMI)")));
-        ogE.addElement(new OptionsGruppeEintrag("Meltagun", getPts("Meltagun (AMI)")));
-        add(o1 = new OptionsZaehlerGruppe(ID, randAbstand, cnt, "", ogE, 4));
-
+        checkBuildaVater();
```

Comment:
Which other call requires the BuildaVater?

Reply:
IMAdeptaSororitasRuestkammer.getSpecialWeapons(buildaVater)